### ISSUE TITLE
DLSS/NGX support?

### ISSUE THREAD

**Issue by trialism** [doitsujin/dxvk#1704]
Hello

 Will dxvk support DLSS 2.0 which was introduced in nvidia 450.57? Would it work at all?

**Comment by K0bin** [comment]
That driver brought the infrastructure for DLSS over to Linux, not DLSS itself.

**Comment by trialism** [comment]
Does that mean we can't use the calls through wine? Shouldn't it be the same API?

**Comment by K0bin** [comment]
DLSS itself is still behind NDA and I don't think the ML model itself or the API for it is available on Linux.

**Comment by trialism** [comment]
If the API is not available then what would be the point of that update? If games can't use it then it's pointless.

**Comment by doitsujin** [comment]
> Does that mean we can't use the calls through wine? Shouldn't it be the same API?

Yes, but someone would still need to write a bridge of some sort and that's just not going to happen as long as the it is NDA'd.

TL;DR not happening any time soon.
